### PR TITLE
Moves "Create Application plan" button into main section

### PR DESCRIPTION
**Pull request by josemigallas**
**What this PR does / why we need it**:

As a first step of improving Application plans index page, move the create button from the table into the main section.

**NEW**
Screenshot 2021-03-16 at 11 14 00

**OLD**
Screenshot 2021-03-16 at 11 17 21

**Which issue(s) this PR fixes** 

Part of [APPDUX-779: [3scale][2.11][LO-prio] Improve list of Application plans in Application plans index page](https://issues.redhat.com/browse/APPDUX-779)

### CODE REVIEW

**Review by thomasmaas — APPROVED** [review]

File: app/views/api/application_plans/_listing.html.slim [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
+  / TODO: use card body and other pf styles instead of .data class
+  / div.pf-c-card__body
+  table#plans.data
+    thead
+      tr
+        th = sortable :name, @new_plan.human_attribute_name(:name)
+        th = sortable :contracts_count, @new_plan.human_attribute_name(:contracts)
+        th = sortable :state, @new_plan.human_attribute_name(:state)
+        th.right.operations colspan="3"
+    = render partial: 'api/plans/table_body', locals: {plans: @plans.not_custom}
```
Comment (hallelujah) [inline app/views/api/application_plans/_listing.html.slim:15]: This is not inside the `<section>` anymore?

Comment (thomasmaas) [reply app/views/api/application_plans/_listing.html.slim:15]: good catch!

File: .circleci/config.yml [hunk 2]
```diff
@@ -242,8 +242,7 @@ commands: # reusable commands with parameters
       - run:
           name: Run cucumber tests
           command: |
-            TEST=$(circleci tests glob "features/**/*.feature" | circleci tests split --split-by=timings)
-            bundle exec cucumber --profile ci ${TEST}
+            bundle exec cucumber --profile ci features
```
Comment (hallelujah) [inline .circleci/config.yml:245]: Why? Do you want to run all the features (really all of them) in all containers each time?
If the `circleci` program cannot find the tests, it still splits them (I think randomly or any other means) but it splits them

**Review by hallelujah — CHANGES_REQUESTED** [review]
Please do not change circleci splitting

File: .circleci/config.yml (hunk 2 above)
Comment (josemigallas) [reply .circleci/config.yml:245]: It is merely an experiment, I need to find the failing cucumber
Screenshot 2021-03-16 at 15 50 08

🚧 in gitmoji means "work in progress" so not final

File: app/views/api/application_plans/_listing.html.slim (hunk 1 above)
Comment (josemigallas) [reply app/views/api/application_plans/_listing.html.slim:15]: Yes it is: https://github.com/3scale/porta/pull/2395/files#diff-27bada7c17b20e1a7c90053b1db234a49dbaa399881190b5d0cbe563bcdb3f50R12
But I think it belongs to the parent file. Section is the container whereas the listing (table) is the content.

File: .circleci/config.yml (hunk 2 above)
Comment (hallelujah) [reply .circleci/config.yml:245]: I showed you how to find the failing cucumber
If you want lets talk about that and screenshrare it

Comment (josemigallas) [reply .circleci/config.yml]: Removed

**Review by hallelujah — APPROVED** [review]
All good!

File: features/step_definitions/api/plan_steps.rb [hunk 3]
```diff
@@ -65,18 +65,18 @@ def new_application_plan_form
...
     fill_in 'System name', with: name
 
-    click_on 'Create Application Plan' # TODO: should be Application Plan
+    click_on 'Create Application Plan'
   end
 
   page.should have_content("Created Application plan #{name}")
 
-  @plan = Plan.find_by_name!(name)
+  @plan = Plan.find_by!(name: name)
```
Comment (hallelujah) [inline features/step_definitions/api/plan_steps.rb:79]: ❤️

**Pull request merged**